hooks/pre_gen_project.py

```diff
@@ -80,3 +80,12 @@
         "You should either use AWS or select a different Mail Service for sending emails."
     )
     sys.exit(1)
+
+if (
+    "{{ cookiecutter.use_docker }}".lower() == "n"
+    and "{{ cookiecutter.use_watchman }}".lower() == "y"
+):
+    print(
+        "Automatic watchman installation is only setup in Docker for now. You can install "
+        "watchman from https://github.com/facebook/watchman/releases and pywatchman from pypi."
```

Comment:
Idk if you want to mention pywatchman from pypi. Instead, say something like "refer to the cookiecutter-django docs for Python installation"

Reply:
True